src/components/pager/css/Pager.css

```diff
@@ -53,6 +53,12 @@
     cursor:default;
 }
 
+/* Pager List Item Conatiner Width */ 
+
+.flc-pager-li-container {
```

Comment:
The name of this class will need to change as a user may use something other than an `<li>` also should consider if they don't have that containing element at all.